src/change.jl

```diff
@@ -0,0 +1,33 @@
+
+"""
+    obs = ChangeObservable(val)
+    obs = ChangeObservable{T}(val)
+
+Like an `Observable`, but only updates watchers when the value changes
+"""
+struct ChangeObservable{T} <: AbstractObservable{T}
```

Comment:
If we use our own new type, shouldn't we just implement the functionality of not updating on the same value in notify, instead of using two observables here?